packages/react-devtools-shared/src/backend/legacy/renderer.js

```diff
@@ -770,11 +782,15 @@ export function attach(
     return {
       id,
 
-      // Hooks did not exist in legacy versions
+      // Does the current renderer support editable hooks and function props?
       canEditHooks: false,
+      canEditFunctionProps: false,
 
-      // Does the current renderer support editable function props?
-      canEditFunctionProps: true,
+      // Does the current renderer support advanced editing interface?
+      canEditHooksAndDeletePaths: false,
+      canEditHooksAndRenamePaths: false,
+      canEditFunctionPropsDeletePaths: false,
+      canEditFunctionPropsRenamePaths: false,
```

Comment:
None of the new reconciler-injected methods will ever exist for legacy React, so these values are always hard-coded to false.